GitSearch/Models/Constants.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  Constants.swift
+//  GitSearch
+//
+//  Created by Vldmr on 8/21/19.
+//  Copyright © 2019 mudritskiy. All rights reserved.
+//
+
+import UIKit
+
+enum Constants {
+    
+    enum ColorScheme {
```

Comment:
It might be better to put all this constants into `UIColor` extension. In that way, you will have uniform access to custom as well as system-supplied colors (e.g., `UIColor.yellow` and `UIColor.mainBlue`).

Reply:
Such extension must be implement in new separated file, or in same with constants?
What is the "best practice"?

Reply:
It definitely deserves its own file :)